internal/hw/amd_linux.go

```
@@ -0,0 +1,191 @@
+			},
+		})
+	}
+	return result, nil
+}
+
+func readKFDProperties(path string) (map[string]uint64, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `file.Close` is not checked

(errcheck)